testsuite/features/build_validation/smoke_tests/smoke_tests.template

```diff
@@ -137,3 +138,16 @@ Feature: Smoke tests for <client>
     When I force picking pending events on "<client>" if necessary
     And I wait at most 600 seconds until event "System reboot scheduled by admin" is completed
     Then I should see a "This action's status is: Completed" text
+
+  Scenario: Install spacecmd from the client tools on the <client>
+    Given I am on the Systems overview page of this "<client>"
```

Comment:
In theory we don't need this first line.
If `Then I should see a "This action's status is: Completed" text` ends in a state where "Software" tab is visible.

Reply:
ok, removing it

i needed it for testing, so i was unsure (i was testing the scenario in isolation)